docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml

```diff
@@ -988,31 +1007,40 @@ thor: []
       </sect3>
 
       <sect3 id="FileAccessCosts">
-        <title>File Access Cost</title>
+        <title>File Access Costs</title>
+<para> For logical files, the costs calculated are based on two types of file access.  </para>
```

Comment:
It should probably be "...there are two types of file costs:".  There is just one type of file access cost (File Access Cost) but there are 2 storage related costs (File Access Cost and File Cost at Rest)